### ISSUE TITLE
npm run-script build not working (on windows?)

### ISSUE THREAD

**Issue by c24-gesundheit-blackfire** [cdaringe/revealer#50]
The build command cannot be executed, because "node_modules" is not found.
I tried it with having grunt and grunt-cli installed locally and globally.

```
> presentations@1.0.0 build C:\Users\xxxxx\code\presentation-doctrine-learnings
> revealer --build dist

Der Befehl "node_modules" ist entweder falsch geschrieben oder
konnte nicht gefunden werden.
Error: Command failed: node_modules/.bin/grunt.cmd css uglify
Der Befehl "node_modules" ist entweder falsch geschrieben oder
konnte nicht gefunden werden.
```

This is in my package.json, otherwise it is a clean new repo:
```
"scripts": {
    "start": "revealer --start",
    "build": "revealer --build dist"
  },
```

**Comment by cdaringe** [comment]
is `C:\Users\xxxxx\code\presentation-doctrine-learning\node_modules` not present?  if not, you'll need to `npm i` or `yarn` first

**Comment by c24-gesundheit-blackfire** [comment]
It is present. It fails after phantomjs is already started.

**Comment by cdaringe** [comment]
or rather, is `C:\Users\xxxxx\code\presentation-doctrine-learning\node_modules\node_modules\.bin\grunt[.cmd]` present?

**Comment by cdaringe** [comment]
when installing `revealjs`, it should place such a file into that directory.  `revealer` just calls into `revealjs`'s `grunt` instance

**Comment by c24-gesundheit-blackfire** [comment]
``node_modules`` inside ``node_modules`` is not present. because I installed grunt manually, the ``grunt.cmd`` is inside ``presentation-doctrine-learning/node_modules/.bin``

**Comment by cdaringe** [comment]
Oops! That was a typo--i only meant one level of node_modules.  I'm AFK ATM, but this could be a simple case of bad path resolution. I did not test on Windows fortunately as I don't have Windows machine accessible.  I hardcoded / in some paths that need to be path.sep instead

**Comment by cdaringe** [comment]
i'm sorry for the inconvenience.  i'll send a patch later.

- bad: https://github.com/cdaringe/revealer/blob/master/src/build.js#L22
- good: https://github.com/cdaringe/revealer/blob/master/src/serve.js#L21

**Comment by Arany** [comment]
@cdaringe Thanks for the help. Could you check out the PR?

**Comment by cdaringe** [comment]
published in `2.1.6`.  try it out, will ya?  run w/ `--verbose` if possible,  in case things go wrong

1. i asked you to test the wrong path! :) . it was actually `node_modules/reveal.js/node_modules/.bin/grunt`, not `node_modules/.bin/grunt`!  found a few more places w/ non windows friendly paths.  sorry :$!
1. you shouldn't need to install grunt locally any longer--it should pick up grunt
1. i also made it such that if you don't already have a `src/index.html`, it adds a the default reveal.js demo

🌮 thx!
